pkg/apis/pipeline/v1beta1/task_validation.go

```diff
@@ -451,15 +452,11 @@ func validateStepArrayUsage(step Step, prefix string, vars sets.String) *apis.Fi
 }
 
 func validateVariables(ctx context.Context, steps []Step, prefix string, vars sets.String) (errs *apis.FieldError) {
-	// validate that the variable name format follows the rules
```

Comment:
could you keep this comment around, and just move it to where the regex is defined?

Reply:
sure! sgtm.